packages/client/hmi-client/src/page/Home.vue

```diff
@@ -72,14 +84,27 @@ const scroll = (direction: 'right' | 'left', event: PointerEvent) => {
 	}
 
 	const marginLeftString =
-		cardListElement.style.marginLeft === '' ? '0' : cardListElement.style.marginLeft;
+		cardListElement.style.marginLeft === '' ? '0.5' : cardListElement.style.marginLeft;
 	const currentMarginLeft = parseFloat(marginLeftString);
 	const changeInRem = direction === 'right' ? -SCROLL_INCREMENT_IN_REM : SCROLL_INCREMENT_IN_REM;
 	const newMarginLeft = currentMarginLeft + changeInRem;
 	// Don't let the list scroll far enough left that we see space before the
 	//	first card.
-	cardListElement.style.marginLeft = `${newMarginLeft > 0 ? 0 : newMarginLeft}rem`;
+	cardListElement.style.marginLeft = `${newMarginLeft > 0 ? 0.5 : newMarginLeft}rem`;
 };
+
+async function createNewProject() {
+	const author = auth.name ?? '';
+	const project = await ProjectService.create(
+		newProjectName.value,
+		newProjectDescription.value,
+		author
```

Comment:
Notes for @YohannParis: we should decide if user attributes should be passed as part of the API input, or to be determined via the auth-token within the server itself.

Reply:
Chatted with @chris-dickson: the `hmi-server` should be doing it.

Reply:
Yeah I thought as much. Let's keep a mental note of that.

Reply:
Yes, and we should add it to `DEVELOPMENT.md`.
I will create a new Issue to fix this problem in a new PR and move on with this one.

Reply:
`hmi-server` just proxies to data service so should this be done there?